contracts/registrar/ZNSRootRegistrar.sol

```diff
@@ -87,13 +90,18 @@ contract ZNSRootRegistrar is
         string calldata tokenURI,
         DistributionConfig calldata distributionConfig
     ) external override returns (bytes32) {
+        // Confirms string values are only [a-z0-9]
+        name.validate();
```

Comment:
why do we do it in this function as well? we will do this twice for every register